boards/arm/efr32mg_sltb004a/doc/index.rst

```diff
@@ -74,6 +74,8 @@ The efr32mg_sltb004a board configuration supports the following hardware feature
 +-----------+------------+-------------------------------------+
 | I2C       | on-chip    | i2c port-polling                    |
 +-----------+------------+-------------------------------------+
+| SPI(M)    | on-chip    | spi port-polling                    |
```

Comment:
What's the significance of the ``(M)`` notation?  Master? Can this just be ``SPI`` or if the ``M`` is important is should be explained?

Reply:
Yes, it means master. Slave-mode is not implemented by this driver. I copied the notation from https://docs.zephyrproject.org/latest/boards/arm/nrf52840_pca10059/doc/index.html#supported-features which contains `SPI (M/S)`.
I do not know what the preferred way for this is. I can also remove the `(M)`.

Reply:
If you think technical folks will know what this means, then it's OK by me.  Otherwise, maybe say ```SPI Master`` to be clear.